config/environments/development.rb

```diff
@@ -5,13 +5,12 @@
     ENV[key.to_s] = value
   end if File.exists?(env_file)
 
-  ENV['DISTRICT_KEY'] = 'somerville'
 
+  ENV['DISTRICT_KEY'] = 'somerville'
   ENV['DISTRICT_NAME'] = 'Localhost Public Schools'
-
   ENV['USE_MOCK_LDAP'] = 'true'
-
   ENV['MOCK_LDAP_PASSWORD'] = 'demo-password'
+  ENV['ENABLE_CLASS_LISTS'] = 'true'
```

Comment:
@alexsoble This is what triggered mentionbot.  This adds in `ENABLE_CLASS_LISTS` as true in development.